docs/abssightline.rst

```diff
@@ -0,0 +1,75 @@
+.. _AbsSightline:
+
+******************
+AbsSightline Class
+******************
+
+.. index:: AbsSightline
+
+Notebooks
+=========
+
+.. toctree::
+   :maxdepth: 1
+
+   Simple Examples <AbsSightline>
+
+Overview
+========
+
+This Class is designed to organize the absorption systems
+along a single sightline.  It may be most commonly used
+for extragalactic sightlines, but it can be applied to the
+ISM as well.
+
+By definition, an AbsSightline is a unique collection of
+absorption components.  The only quantities required
+ to define the AbsSightline are its coordinates on the sky.
+
+
+Instantiation
+=============
+
+The AbsSightline Class may be instantiated in a few ways.
+The default sets the properties listed above::
+
+	abssl = GenericAbsSightline((10.0*u.deg, 45*u.deg))
+
+More commonly, one will instantiate with one
+a set of components::
+
+    lya = AbsLine('HI 1215')
+    lya.attrib['z'] = 2.3
+    lyb = AbsLine(1025.7222*u.AA, z=lya.attrib['z'])
+    lyb.limits.set([-300.,300.]*u.km/u.s)  # vlim
+    abscomp = AbsComponent.from_abslines([lya,lyb])
```

Comment:
If I do not also set lya.limits, I get the following warning and the resulting abscomp has [0,0] velocity limits:

Input absline with wrest=1025.72 Angstrom at z=2.300 does not match component rules. Not appending
Absline velocities lie beyond component
 Set chk_vel=False to skip this test.